CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # v 2.9.0 (?)
 Changes in this release:
+- Adds a new resource entity that raises an IgnoreResourceException when buildings its id
```

Comment:
I'm not 100% sure whether a changelog entry is relevant here or not

Reply:
I would keep it. I agree that it's unlikely that our customers will use this feature, but it's a new feature anyway.